src/pages/Employees.jsx

```diff
@@ -1,5 +1,49 @@
+import React, { useState } from "react";
+import { AgGridReact } from "ag-grid-react";
+import "ag-grid-community/styles/ag-grid.css";
+import "ag-grid-community/styles/ag-theme-alpine.css";
+
 const Employees = () => {
-  return <div>Employees</div>;
+  const [rowData] = useState([
+    {
+      serialNo: 1,
+      employeeType: "Clinician",
+      clinicianName: "PATRICIA EPIE",
+      clinicianEmailId: "epipat.pe@gmail.com(CHCC0008)",
+      assigned: 6,
+      onGoing: 1,
+      completed: 0,
+      reOpen: 0,
+      total: 7,
+      lastLogin: "17-May-2023 17:36:36",
+    },
+  ]);
+
+  const [columnDefs] = useState([
```

Comment:
No need to have column definitions stored in state. Just a normal variable (constant) is fine.